source/fetch.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const { generateDigestAuthHeader } = require("./auth.js");
+const { merge } = require("./merge.js");
+
+const digest = { nc: 0, algorithm: "md5" };
+
+let hasDigestAuth = false;
```

Comment:
Having these as statics doesn't allow for multiple webdav-client instances.. Perhaps we could somehow tie all of this to the webdav instance out of the factory?

Reply:
See https://github.com/perry-mitchell/webdav-client/pull/152/commits/8c52e4ffd8d7265abcc4f01c6e956ef861cb80c8#diff-4970d419691cca822418c65c1fe218ff